### ISSUE TITLE
AUTO IGNITION TEMPERATURE units incorrect in .out file

### ISSUE THREAD

**Issue by rmcdermo** [firemodels/fds#10727]
When I specify the auto-ignition temperature on the REAC line, the units are supposed to be in C.  They are converted to K in the .out file, but the numeric values looks to still be in C.  Here is a simple example:

```
&HEAD CHID='simple_1reac', TITLE='reproduce 1 simple reactions'/
&MESH IJK=4,4,4, XB=0,1,0,1,0,1/
&TIME T_END=.01/
&REAC FUEL='METHANE', AUTO_IGNITION_TEMPERATURE=-273 /
&TAIL /
```

To me, this conversion is confusing to begin with.  I like being able to take information from the .out and put it back in the input file without more conversions.  Can we make the .out use (C)?

**Comment by rmcdermo** [comment]
The units are not the only problem.  If you enter 0 C then it gets converted to -273 K.

**Comment by rmcdermo** [comment]
I'm fixing this.

**Comment by rmcdermo** [comment]
Should be fixed with PR #10729
